plugins/modules/source_control/gitlab/gitlab_runner.py

```diff
@@ -57,6 +57,12 @@
       - The registration token is used to register new runners.
     required: True
     type: str
+  owned:
+    description:
+      - Will search only owned runners when searching for existing, when false admin token required
```

Comment:
```suggestion
      - Searches only owned runners when searching for existing, when false admin token required.
```
Please, use the present tense, this is a doc requirement, and a final dot.